groups/sig-contributor-experience/groups.yaml

```diff
@@ -69,6 +69,26 @@ groups:
       - pal.nabarun95@gmail.com
       - priyankasaggu11929@gmail.com
 
+  - email-id: kubernetes-github-managment-alerts@kubernetes.io
+    name: kubernetes-github-managment-alerts
+    description: |-
+      Open mailing list for alerts from Kubernetes GitHub management applications. All other posts will be moderated. Please do not attempt to reply to this list.
+
+      For more information, refer https://git.k8s.io/community/github-management
+    owners:
+      - cblecker@gmail.com
+    managers:
+      - killen.bob@gmail.com
+      - madhav.jiv@gmail.com
+      - nikitaraghunath@gmail.com
+      - pal.nabarun95@gmail.com
+      - priyankasaggu11929@gmail.com
+    settings:
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      ReconcileMembers: "true"
```

Comment:
```suggestion
      WhoCanJoin: "ANYONE_CAN_JOIN"
      WhoCanViewGroup: "ANYONE_CAN_VIEW"
      WhoCanPostMessage: "ANYONE_CAN_POST"
      MessageModerationLevel: "MODERATE_ALL_MESSAGES"
      WhoCanViewMembership: "ALL_MANAGERS_CAN_VIEW"
      WhoCanModerateMembers: "OWNERS_AND_MANAGERS"
      WhoCanModerateContent: "OWNERS_AND_MANAGERS"
      MembersCanPostAsTheGroup: "false"
      ReconcileMembers: "false"
```

Reply:
Following three, I found [here](https://github.com/kubernetes/k8s.io/blob/40aee2bfac866db1f881a9842fbc5ede7ef93bd8/groups/service.go#L459-L463), are set as default settings, so dropped them:

```
      WhoCanViewMembership: "ALL_MANAGERS_CAN_VIEW"
      WhoCanModerateMembers: "OWNERS_AND_MANAGERS"
      WhoCanModerateContent: "OWNERS_AND_MANAGERS"
```

Added: `MessageModerationLevel: "MODERATE_ALL_MESSAGES"`.

Should I do the same cleanup in other MLs too? (in a separate commit)